maraboupy/MarabouCore.cpp

```diff
@@ -114,6 +115,18 @@ void addMaxConstraint(InputQuery& ipq, std::set<unsigned> elements, unsigned v){
     ipq.addPiecewiseLinearConstraint(m);
 }
 
+void addSoftmaxConstraint(InputQuery& ipq, std::list<unsigned> inputs,
```

Comment:
Use List instead of std::list?

Reply:
Also, add spaces to the parameter lists
void addSoftmaxConstraint( InputQuery& ipq, std::list<unsigned> inputs,
                                             ^

Reply:
Since we are going to call this function in python using pybind, I think we have to use std::list<unsigned>

Reply:
The format in MarabouCore.cpp is completely different from the rest of the code base btw. Probably need a clang pass to fix those.